msp/compliance/orgs_and_cloud_accounts_report/orgs_and_cloud_accounts_report.pt

```diff
@@ -0,0 +1,278 @@
+name "Orgs and Clouds Vendor Accounts"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy generates a list of cross organization Cloud Vendor Accounts connected to Flexera Optima based on
+the bill connection settings for Azure and Google, as well as full list of AWS accounts under the payer account connected for each Flexera Organization.\n
+See [README] for more details"
+long_description ""
+severity "low"
+category "Compliance"
+info(
+  version: "1.0",
+  provider: "Flexera Optima",
+  service: "",
+  policy_set: ""
```

Comment:
Please add a new field to the info section:  publish: false, to skip/avoid publishing to the catalog.

Reply:
Done